projects/public-app-getting-started-template/src/app/public-app.json

```diff
@@ -2,7 +2,7 @@
   "name": "Get started with public apps",
   "uid": "get-started-public-app",
   "description": "An example to demonstrate how to build a public app with developer projects.",
-  "allowedUrls": ["https://api.hubapi.com"],
+  "allowedUrls": ["https://api.yourplatform.com"],
```

Comment:
I think `https://api.example.com` would be a bit more conventional, but we don't have to overthink it.